src/output.rs

```diff
@@ -87,6 +88,20 @@ impl OutputType {
     }
 }
 
+// Since someone could set PAGER to less without arg -R that bat needs,
+// ignore flags for less and change flags to -FRX.
+// If a user wants an env variable with flags unchanged, he/she should use
+// BAT_PAGER.
+fn add_default_flags_to_less(pager: String) -> String {
+    if let Ok(flags) = shell_words::split(&pager) {
+        if flags.first() == Some(&String::from("less")) {
```

Comment:
This does not work if the first argument is `/usr/bin/less`, for example. See the `is_less` variable example in the same file.